### ISSUE TITLE
Actions defined using aria-label cannot be queried on insights

### ISSUE THREAD

**Issue by aydtsang** [PostHog/posthog#11548]
## Bug description
When defined on the toolbar, it will say that it matches the element. However, in the heatmap it will not show up with the events that should be associated with it and the insights page will fail to complete the query.

<img width="250" alt="Screen Shot 2022-08-30 at 8 30 43 AM" src="https://user-images.githubusercontent.com/16506503/187437425-580aed0c-7d4b-49a2-a161-a3b37efe8d6d.png">

<img width="649" alt="Screen Shot 2022-08-30 at 8 26 33 AM" src="https://user-images.githubusercontent.com/16506503/187437375-6c5842dc-1d71-4373-8914-79d068245c7e.png">

## How to reproduce

1. Make an action with properties CSS selector [aria-label={{label}}]
2. Try to plot it as an insight

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: please provide
